utils/check_license/check-license.c

```diff
@@ -45,6 +45,10 @@
 #include <fcntl.h>
 #include <unistd.h>
 #include <ctype.h>
+#ifdef __FreeBSD__
+#include <sys/types.h>
+#include <sys/stat.h>
```

Comment:
I think we can't just always include these two headers because of Windows.
If we would have at least empty version under `src/windows`, then we wouldn't need another ifdef here either.

What would be even better, is to change this source to use ISO C stuff: fopen, fread, fwrite, fclose instead of open, read, write, close.

There should be no need for POSIX for checking some text in text files.